ts/supabase/README.md

```diff
@@ -0,0 +1,133 @@
+# Supabase React SDK + Encore App Example
+
+This is an example of how to implement user authentication using [Supabase](https://supabase.com/) together with an Encore app, and make a API call to an Encore backend!
+
+
+## Cloning the example
+
+### Prerequisite: Installing Encore
+
+If this is the first time you're using Encore, you first need to install the CLI that runs the local development
+environment. Use the appropriate command for your system:
+
+- **macOS:** `brew install encoredev/tap/encore`
+- **Linux:** `curl -L https://encore.dev/install.sh | bash`
+- **Windows:** `iwr https://encore.dev/install.ps1 | iex`
+
+When you have installed Encore, run to clone this example:
+
+```bash
+encore app create my-app --example=ts/supabase
+```
+
+## Supabase Credentials
+
+1. Create a Supabase account if you haven't already. Then, create a new project in the Supabase dashboard.
+
+2. Go to the *Project Settings* > *API* page for your project. Copy the "Project URL" and "Project API Key".
+
+3. In `frontend/.env` file, replace the values for `VITE_SUPABASE_URL` and `VITE_SUPABASE_ANON_KEY` with the values from your Supabase dashboard.
+
+4. The `Project API Key` is sensitive and should not be hardcoded in your code/config. Instead, you should store that as an [Encore secret](https://encore.dev/docs/primitives/secrets).
+
+From your terminal (inside your Encore app directory), run:
+
+```shell
+$ encore secret set --prod SUPABASE_URL
+```
+
```

Comment:
```suggestion
NOTE: This will use the same `service_role` for both development and production environments. You might want to create separate Supabase project for your different environments. 

```